File: Web/ClientApp/src/app/_models/enums.ts
export enum TypeOfMessage {
  chat = 1,
  server = 2,
  spectators = 3
}
export enum ChatDestination {
  all = 1,
  tournament = 2,
  game = 3
}

export enum CardColor {
  blue = 1,
  green = 2,
  red = 3,
  yellow = 4,
  wild = 5
}

export enum CardValue {
  zero = 0,
  one = 1,
  two = 2,
  three = 3,
  four = 4,
  five = 5,
  six = 6,
  seven = 7,
  eight = 8,
  nine = 9,
  reverse = 10,
  skip = 11,
  drawTwo = 12,
  drawFour = 13,
  changeColor = 14,
  blackHole = 15,
  discardWildCards = 16,
  stealTurn = 17,
  swapHands = 18,
  doubleEdge = 19,
  discardColor = 20,
  handOfGod = 21,
  judgement = 22,
  unitedWeFall = 23,
  paradigmShift = 24,
  deflect = 25,
  inspectHand = 26,
  graveDigger = 27,
  russianRoulette = 28,
  roulette = 29,
  duel = 30,
  keepMyHand = 31,
  charity = 32,
  blackjack = 33,
  fairPlay = 34,
  tricksOfTheTrade = 35,
  discardNumber = 36,
  theLastStand = 37,
  magneticPolarity = 38,
  fortuneTeller=39,
  doubleDraw=40,
  poison=41,
  randomColor=42,
  promiseKeeper=43,
  gambling = 44,
  copyCat = 45
}

export enum Direction {
  right = 1,
  left = 2
}
export enum GameType {
  normal = 1,
  specialWildCards = 2
}

export enum PlayersSetup {
  individual = 1,
  teams = 2
}

File: Common/Enums/Enums.cs
namespace Common.Enums
{
    public enum CardColor
    {
        Blue = 1,
        Green = 2,
        Red = 3,
        Yellow = 4,
        Wild = 5
    }

    public enum CardValue
    {
        Zero = 0,
        One = 1,
        Two = 2,
        Three = 3,
        Four = 4,
        Five = 5,
        Six = 6,
        Seven = 7,
        Eight = 8,
        Nine = 9,
        Reverse = 10,
        Skip = 11,
        DrawTwo = 12,
        DrawFour = 13,
        ChangeColor = 14,
        BlackHole = 15,
        DiscardWildCards = 16,
        StealTurn = 17,
        SwapHands = 18,
        DoubleEdge = 19,
        DiscardColor = 20,
        HandOfGod = 21,
        Judgement = 22,
        UnitedWeFall = 23,
        ParadigmShift = 24,
        Deflect = 25,
        InspectHand = 26,
        GraveDigger = 27,
        RussianRoulette = 28,
        Roulette = 29,
        Duel = 30,
        KeepMyHand = 31,
        Charity = 32,
        Blackjack = 33,
        FairPlay = 34,
        TricksOfTheTrade = 35,
        DiscardNumber = 36,
        TheLastStand = 37,
        MagneticPolarity = 38,
        FortuneTeller = 39,
        DoubleDraw = 40,
        Poison = 41,
        RandomColor = 42,
        PromiseKeeper = 43,
        Gambling = 44,
        CopyCat = 45,
        RobinHood= 46,
        Handcuff=47,
        Silence=48,
        Assassinate=49,
        SummonWildcard=50,
        DevilsDeal=51,
        KingsDecree=52,
        QueensDecree=53,
        Surprise=54

    }

    public enum Direction
    {
        Right = 1,
        Left = 2
    }
    public enum GameType
    {
        Normal = 1,
        SpecialWildCards = 2
    }
    public enum PlayersSetup
    {
        Individual = 1,
        Teams = 2
    }

    public enum TypeOfMessage
    {
        Chat = 1,
        Server = 2,
        Spectators = 3
    }
    public enum ChatDestination
    {
        All = 1,
        Tournament = 2,
        Game = 3
    }

    public enum ChatMessageIntention
    {
        Normal = 1,
        Buzz = 2
    }

}